src/components/ShareList.js

```diff
@@ -39,22 +39,22 @@ const ShareList = props => {
     <div>
       <form onSubmit={handleSubmit}>
         <label htmlFor="sharedToken">
-          Join a list:
+          Token:
           <input
             type="text"
             name="sharedToken"
             id="sharedToken"
             onChange={event => setUserToken(event.target.value)}
           />
         </label>
-        <input type="submit" value="Submit" />
+        <input type="submit" value="Join List" />
       </form>
       <div>
         {isError ? (
-          <p>
-            'We couldn't find that shopping list. Please try again or create a
-            new list.'
-          </p>
+          <React.Fragment>
```

Comment:
Another way to write a React.Fragment is: `<></>` instead of `<React.Fragment></React.Fragment>`

Reply:
I just learned about `<React.Fragement>`. So thanks for teaching me a shorthand! It's good to know.